packages/server/src/integrations/mysql.ts

```diff
@@ -317,8 +318,10 @@ class MySQLIntegration extends Sql implements DatasourcePlus {
         }
         if (!tables[tableName]) {
           tables[tableName] = {
+            type: "table",
```

Comment:
Will we need to migrate all the existing ds?

Reply:
This will be handled as part of enrichment of the tables when they are retrieved from the API/saved. Essentially we'll be migrating in place.

I've tested this - we already depended on enrichment for `internal/external` type value - so its not much different what we're doing now, just more consistent.